### ISSUE TITLE
False positives for "exported" rule

### ISSUE THREAD

**Issue by entrope** [mgechev/revive#229]
**Describe the bug**
revive reports "comment on exported type T should be of the form "T ..." (with optional leading article)" when the comment on that type is of the form "/* T ... */" (with a leading space).  Likewise for exported functions, and presumably ValueSpec entries as well.

**To Reproduce**
1. I updated revive `go get -u github.com/mgechev/revive` (commit fa5acbc1f03, from 31 Aug)
2. I run it with the following flags, configuration file, and Go source file:

```shell
revive -config exported.toml exported.go
```

```toml
[rule.exported]
```

```golang
package exported

/* Exported is an exported structure. */
type Exported struct {
	p int
}
```

**Expected behavior**
I expected no warning.  If I change the comment style to //, the warning does not occur.

**Desktop (please complete the following information):**
 - OS: Ubuntu 19.04
 - Tested with go1.10.4 and go1.13.

**Additional context**
It is easy to fix with strings.TrimSpace() in the obvious spots in exported.go, but I could not figure out how to add a regression test, so I did not file this as a PR.  I would be happy to do that, but the problem is small enough that it may be easier to do it than to walk me through adding a suitable regression test.

**Comment by chavacava** [comment]
Hi @entrope, thanks for filling the issue.

I've created the PR #231. There you can check if fixes the problem and you can also see how to add regression tests for a rule.
